src/Node/Api.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the KongAdminApi package.
+ *
+ * (c) Unikorp <https://github.com/unikorp>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Unikorp\KongAdminApi\Node;
+
+use Unikorp\KongAdminApi\AbstractNode;
+use Unikorp\KongAdminApi\Document\Api as Document;
+
+/**
+ * api
+ *
+ * @author VEBER Arnaud <https://github.com/VEBERArnaud>
+ */
+class Api extends AbstractNode
+{
+    /**
+     * add api
+     *
+     * @param \Unikorp\KongAdminApi\Document\Api $document
```

Comment:
Wrong PHPdoc here (and in each method, in each node type), it should be `@param Document $document`